### DISCUSSION TITLE
[HELP] How to implement the the code-editing-mode like online demo?

### DISCUSSION THREAD

**Discussion by langsz** [Milkdown/milkdown#436] [General]
Hi,
I'm learning milkdown. I want to implement the editor have two mode. BUt I don't know how to open the code editing mode.  Can you help me?

**Comment by Saul-Mirone** [comment]
You can find the source code here:
https://github.com/Saul-Mirone/milkdown/blob/main/gh-pages/component/Demo/Demo.tsx
